src/main/java/nextstep/subway/path/domain/PathFinder.java

```diff
@@ -0,0 +1,60 @@
+package nextstep.subway.path.domain;
+
+import nextstep.subway.line.domain.Lines;
+import nextstep.subway.line.domain.Section;
+import nextstep.subway.station.domain.Station;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+import java.util.Objects;
+
+public class PathFinder {
+    private static final String MUST_NOT_BE_NULL = "null일 수 없습니다.";
+
+    private WeightedMultigraph<Station, DefaultWeightedEdge> graph;
```

Comment:
final 을 사용해보면 어떨까요? 😄 
```suggestion
    private final WeightedMultigraph<Station, DefaultWeightedEdge> graph;
```

Reply:
넵 final로 변경하겠습니다~